docs/pages/box.js

```diff
@@ -1,258 +1,70 @@
-        },
-        { name: 'paddingX', description: 'Horizontal padding (left/right)' },
-        { name: 'paddingY', description: 'Vertical padding (top/bottom)' },
-      ].map(({ description = '', name }) => ({
-        description,
-        name,
-        type: '0 .. 12',
-        defaultValue: 0,
-        responsive: true,
-      })),
-      {
-        name: 'position',
-        type: `"static" | "absolute" | "relative" | "fixed"`,
-        defaultValue: 'static',
-        description: 'See the [absolute positioning](#Absolute-positioning) variant for more info.',
-      },
-      {
-        name: 'ref',
-        type: "React.Ref<'div'>",
-        description:
-          'Forward the ref to the underlying div element. See the [using as a ref](#Using-as-a-ref) variant for more info.',
-      },
-      {
-        name: 'role',
-        type: 'string',
-        description:
-          'Used to designate the Box as a type of element or landmark using ARIA roles. See the [Accessibility guidelines](#Using-role) to learn more about using `role`.',
-      },
-      {
-        name: 'rounding',
-        type: `"pill" | "circle" | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8`,
-        description: 'See the [rounding](#Rounding) variant for more info.',
-      },
-      {
-        name: 'userSelect',
-        type: `"auto" | "none"`,
-        defaultValue: 'auto',
-        description: `Controls whether or not user can select text`,
-      },
-      {
-        name: 'wrap',
-        type: 'boolean',
-        defaultValue: false,
-        description: `By default, flex items will all try to fit onto one line. You can change that and allow the items to wrap onto multiple lines, from top to bottom.`,
-      },
-      {
-        name: 'zIndex',
-        type: 'interface Indexable { index(): number; }',
-        description: `An object representing the zIndex value of the Box. See the [Z-Index](#Z-Index) variant for more info.`,
-      },
-    ]}
-  />,
-);
-
-card(
-  <MainSection name="Best practices">
-    <MainSection.Subsection columns={2}>
-      <MainSection.Card
-        cardSize="md"
-        type="do"
-        description={`
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import docgen, { type DocGen } from '../components/docgen.js';
+
+const ignoredProps = [
+  'smColumn',
+  'mdColumn',
+  'lgColumn',
+  'smDirection',
+  'mdDirection',
+  'lgDirection',
+  'smDisplay',
+  'mdDisplay',
+  'lgDisplay',
+  'smMargin',
+  'mdMargin',
+  'lgMargin',
+  'smMarginBottom',
+  'mdMarginBottom',
+  'lgMarginBottom',
+  'smMarginEnd',
+  'mdMarginEnd',
+  'lgMarginEnd',
+  'smMarginStart',
+  'mdMarginStart',
+  'lgMarginStart',
+  'smMarginTop',
+  'mdMarginTop',
+  'lgMarginTop',
+  'smPadding',
+  'mdPadding',
+  'lgPadding',
+  'smPaddingX',
+  'mdPaddingX',
+  'lgPaddingX',
+  'smPaddingY',
+  'mdPaddingY',
+  'lgPaddingY',
+];
+
+export default function BoxPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
```

Comment:
Most of this content is the same, but I copyedited here and there. Not really any major changes.